package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -736,11 +849,20 @@ dnsmasq_start()
 	config_foreach filter_dnsmasq mxhost dhcp_mx_add "$cfg"
 	echo >> $CONFIGFILE_TMP
 
-	config_get odhcpd_is_active odhcpd maindhcp
-	if [ "$odhcpd_is_active" != "1" ]; then
-		config_foreach filter_dnsmasq dhcp dhcp_add "$cfg"
+
+	if [ "$DNSMASQ_DHCP_VER" -gt 4 ] ; then
+		# Enable RA feature for when/if it is constructed,
+		# and RA is selected per interface pool (RA, DHCP, or both),
+		# but no one (should) want RA broadcast in syslog
+		config_foreach dhcp_add dhcp
+		xappend "--enable-ra"
+		xappend "--quiet-ra"
```

Comment:
Option `--enable-ra` will not transmit RA unless `--dhcp-range=.../6...` is assigned to a respective interface. Unless you have another RA/DHCP server binding to interfaces, it won't cause trouble.